### PR TITLE
Dashboard - Add baseUrl and simplify paths in components 

### PR DESCRIPTION
**Pull request by psmolin**
### Before this PR

`import { useTheme } from '../../../../../theme'`

### After this PR

`import { useTheme } from 'theme'`

### CODE REVIEW

File: packages/bootstrap/src/components/icons/Handover.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+
 import { Icon, IconProps } from './Icon';
-import { useTheme } from '../../theme';
+import { useTheme } from 'theme';
```
Comment (i-kushlianski) [inline packages/bootstrap/src/components/icons/Handover.tsx]: import order should be fixed too

File: packages/bootstrap/src/utils/actions.spec.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
-import defaultTheme from '../theme';
-import { clickable } from './actions';
 import { DefaultTheme, CSSObject } from 'styled-components';
-import { HiroColors } from '../types';
+
+import defaultTheme from 'theme';
+import { HiroColors } from 'types';
+import { clickable } from './actions';
```
Comment (i-kushlianski) [inline packages/bootstrap/src/utils/actions.spec.ts]: space between import groups

File: packages/bootstrap/src/utils/flex.spec.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { CSSObject } from 'styled-components';
 
-import { createTheme } from '../theme';
+import { createTheme } from 'theme';
```
Comment (i-kushlianski) [inline packages/bootstrap/src/utils/flex.spec.ts]: pls check other places where we would need spaces between import groups

**Comment by psmolin** [comment]
@i-kushlianski Thanks for the feedback, could you please give it a second look?

File: packages/bootstrap/src/components/dashboard/shared/SessionList/SessionItemDetails.tsx [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 
+import { SiteMap } from 'components/icons';
+
 import { OriginNodeData, SessionData } from '../../types';
```
Comment (i-kushlianski) [inline packages/bootstrap/src/components/dashboard/shared/SessionList/SessionItemDetails.tsx]: from `'types'`

**Pull request merged**